Jenkinsfile_nightly

```diff
@@ -97,18 +97,18 @@ withNightlyPipeline(type, product, component) {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/integration/**/*'
   }
 
-  afterSuccess('build') {
+  afterAlways('build') {
     stage("integration tests") {
       sh "./gradlew integration"
       steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/integration/**/*'
     }
   }
 
-  afterSuccess('fullFunctionalTest') {
+  afterAlways('fullFunctionalTest') {
```

Comment:
Can you add the following to the two `functionalTest` actions to make the reports easier to find.

``` 
      publishHTML target: [
        allowMissing         : true,
        alwaysLinkToLastBuild: true,
        keepAll              : true,
        reportDir            : "target/cucumber/functional-html-reports",
        reportFiles          : "overview-features.html,overview-failures.html,",
        reportTitles         : "Features,Failures",
        reportName           : "Functional Test Report"
      ]
```